src/components/Organisms/NavBar/NavBar.component.tsx

```diff
@@ -0,0 +1,31 @@
+import Box from '@mui/material/Box';
+import Stack from '@mui/material/Stack';
+import React from 'react';
+import { Logo } from '../../Atoms/Logo';
+
+const NavBar = () => {
+  return (
+    <Box
+      width={1}
+      position='sticky'
+      bgcolor='#f5f5f5'
+      top={0}
+      zIndex={100}
+      borderColor='#D9D9D9'
+      borderBottom={2}
+    >
+      <Stack
+        direction='row'
+        alignItems='center'
+        justifyContent='center'
+        width={1}
+        mx='auto'
+        maxWidth={1100}
+      >
+        <Logo />
```

Comment:
@siroshan  We can call this an `organism` if we pass `children` instead of `<Logo />` here—so there will be a `NavbarTemplate` which uses this component to build the final output by passing what's inside it!